packages/core/plugin.schema.json

```diff
@@ -1840,7 +1840,7 @@
               "type": "string"
             },
             "type": {
-              "type": "string"
+              "$ref": "#/components/schemas/CredentialStatusType"
```

Comment:
I think the plugin schemas need to be regenerated.
Please run `yarn build` locally and commit the new results.